packages/core/src/VNodes/AbstractNode.ts

```diff
@@ -28,6 +28,13 @@ export abstract class AbstractNode extends EventMixin {
      */
     get editable(): boolean {
         if (this._editable === false) return false;
+        const formats = this.modifiers.filter(mod => 'modifiers' in mod);
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const lastFormatWithContentEditable = formats.reverse().find(format => hasContentEditable(format as any));
```

Comment:
C'est quoi ces `any` ? Je suis sûr qu'il y a moyen de s'en passer.

Reply:
Oui en fait je devrais prendre `VNode | Modifier` puis tester `'modifiers' in nodeOrModifier` dans `hasContentEditable` et `isContentEditable`.